driver-embedded-android/build.gradle

```diff
@@ -64,6 +64,9 @@ android {
         release {
             minifyEnabled false
         }
+        debug {
```

Comment:
Just confirming we want to keep this and it wasn't meant to be temporary while you debugged.

Reply:
Ah that can now be removed.